content/news/023/index.md

```diff
@@ -91,6 +91,56 @@ making of [video](https://youtube.com/watch?v=3TOEZ7krhvI) on YouTube.
 
 ## Library & Tooling Updates
 
+### [backroll-rs] and [GGRS]
+
+[backroll-rs] ([Discord](https://discord.gg/VuZhs9V),
+[crates.io](https://crates.io/crates/backroll)) by [@james7132] and
+[GGRS]([crates.io](https://crates.io/crates/ggrs)) by [@g_schup] are pure
+Rust implementations of the [GGPO] rollback networking library.
+
+![Evo Moment 37](rollback.jpg)
+_[Evo Moment 37](https://www.youtube.com/watch?v=JzS96auqau0):
+Only offline or with rollback!_
+
+Rollback networking is a peer-to-peer network technique designed to hide
+network latency in fast-paced games with precise inputs. Traditional techniques
+account for network transmission time by delaying the game execution, resulting
+in a sluggish game-feel. Rollback uses input prediction and speculative
+execution instead. Upon receiving inputs from remote clients, resimulation of
+incorrect game states occurs. This allows for gameplay that "feels just
+like offline". The open source standard for rollback netcode [GGPO] is used
+in successful games like Skullgirls, Guilty Gear XX Accent Core +R or
+Fightcade. For further explanation about rollback,
+[click here](https://ki.infil.net/w02-netcode.html).
+
+Two projects in Rust were independently created to provide a working
+implementation as well as helpful resources for developers.
+backroll-rs features an added abstraction for the transportation layer and
+also provides a bevy plugin, [bevy-backroll]. GGRS replaces the
+C-style callback API of GGPO with a simpler, more understandable control flow.
+The authors of both libraries recommend backroll-rs for developement,
+as it is currently more actively collaborated on.
+GGRS is recommended as a learning recource and entry point,
+with a plethora of internal documentation and explanation.
+
+The main requirement to make use of both presented libraries is determinism
+in your game execution. Resimulation requires that the result of progressing
+the game state depending on the given inputs yield the exact same results
+every time. Additionally, you need to be able to load, save and progress
+your gamestate without rendering the outcome.
+
+If you are interested in integrating rollback networking into your game or
+just want to chat with other rollback developers (not limited to Rust),
+check out the [GGPO Developers Discord]!
+
+[backroll-rs]: https://github.com/HouraiTeahouse/backroll-rs
+[bevy-backroll]: https://github.com/HouraiTeahouse/backroll-rs/tree/main/bevy_backroll
+[GGPO Developers Discord]: https://discord.gg/8FKKhCRCCE
+[GGRS]: https://github.com/gschup/ggrs
+[GGPO]: https://www.ggpo.net/
+[@g_schup]: https://twitter.com/
```

Comment:
This is just a link to the twitter homepage. Pretty sure this should be filled out with an actual profile link.

Reply:
good catch!